web/src/classic/core/Crust/Widgets/WidgetAlignSystem/types.ts

```diff
@@ -18,7 +18,7 @@ export type WidgetArea = {
   align?: Alignment;
   padding?: WidgetAreaPadding;
   widgets?: InternalWidget[];
-  gap?: number;
+  gap?: number | any;
```

Comment:
Why is this `any` needed? `any` typically is never necessary and also removes all type safety.

Reply:
yes I know , but actually it is a weird error ( I didn't actually work on these files ) and you all push prs without this change and without this error so I will make it as it was and investigate this error

Reply:
After investigate this error appears after generate schema due to a previous change in backend (making gap mandatory ) and it didn't appear earlier because it seems we didn't generate schema since this change.so to fix that I will make it in backend (number | null )

Reply:
please check this 5986502e8a3d12413413058519d69a070981eb40